### PR TITLE
CI: test lower-bound PyTorch version with Python 3.6

### PR DESCRIPTION
**Pull request by danieldk**
Draft: fails without #610, plus I am not sure if the syntax is valid yet :).

### CODE REVIEW

**Comment by adrianeboyd** [comment]
Let's do the hackier version of this for now, like this:

https://github.com/explosion/spacy-transformers/blob/bebb33cc6929a4b6b37c2488b16b82bf9cc4b082/azure-pipelines.yml#L60-L63

File: azure-pipelines.yml [hunk 1]
```diff
@@ -17,6 +17,10 @@ jobs:
 - job: 'Test'
   variables:
     NOTEBOOK_KERNEL: "thinc-notebook-tests"
+    ${{ if eq(variables['python.version'], '3.6') }}:
+      PYTORCH_VERSION: "1.5.0"
+    ${{ if ne(variables['python.version'], '3.6') }}:
+      PYTORCH_VERSION: "1.11.0"
```
Comment (adrianeboyd) [inline azure-pipelines.yml:23]: I'm not sure `python.version` is defined at this point?. (But I have never understood all the variable substitution details, it's all trial and error.)

**Comment by adrianeboyd** [comment]
This is passing without torch installed, the substitution isn't working as intended.

**Review by svlandeg — COMMENTED** [review]
`master` has changed significantly since this PR was opened, so probably best to merge in the current branch & double check merge conflicts 🙏

**Comment by danieldk** [comment]
> `master` has changed significantly since this PR was opened, so probably best to merge in the current branch & double check merge conflicts 🙏

Since this is a tiny PR, I'll close it and open a new one.

**Pull request closed without merging**